repository/j.json

```diff
@@ -975,6 +975,21 @@
 			]
 		},
 		{
+			"name": "JSON2Go",
+			"details": "https://github.com/everCyan/JSON2Go",
+			"labels": [
```

Comment:
Labels are meant for grouping related packages. Please change your labels to `"labels": ["json"],`